shell/common/rasterizer.cc

```diff
@@ -629,7 +627,6 @@ RasterStatus Rasterizer::DrawToSurfaceUnsafe(
     FireNextFrameCallbackIfPresent();
 
     if (surface_->GetContext()) {
-      TRACE_EVENT0("flutter", "PerformDeferredSkiaCleanup");
```

Comment:
IME this event never shows anything interesting. We should instead just use skia tracing when needed to find out what's going on here.